app/src/main/java/de/danoeh/antennapod/fragment/AudioPlayerFragment.java

```diff
@@ -98,44 +83,30 @@ public View onCreateView(@NonNull LayoutInflater inflater,
                              @Nullable ViewGroup container,
                              @Nullable Bundle savedInstanceState) {
         super.onCreateView(inflater, container, savedInstanceState);
-        View root = inflater.inflate(R.layout.audioplayer_fragment, container, false);
-        toolbar = root.findViewById(R.id.toolbar);
-        toolbar.setTitle("");
-        toolbar.setNavigationOnClickListener(v ->
+        viewBinding = AudioplayerFragmentBinding.inflate(getLayoutInflater());
+
+        viewBinding.toolbar.setTitle("");
+        viewBinding.toolbar.setNavigationOnClickListener(v ->
                 ((MainActivity) getActivity()).getBottomSheet().setState(BottomSheetBehavior.STATE_COLLAPSED));
-        toolbar.setOnMenuItemClickListener(this);
+        viewBinding.toolbar.setOnMenuItemClickListener(this);
 
         ExternalPlayerFragment externalPlayerFragment = new ExternalPlayerFragment();
         getChildFragmentManager().beginTransaction()
-                .replace(R.id.playerFragment, externalPlayerFragment, ExternalPlayerFragment.TAG)
+                .replace(R.id.playerFrag, externalPlayerFragment, ExternalPlayerFragment.TAG)
```

Comment:
Why is this needed?

Reply:
didn't get what you mean.

Reply:
Sorry, I meant the renaming from `playerFragment` to `playerFrag`. I think it would be better to keep the full name.